src/styles/GlobalStyle.js

```diff
@@ -1,7 +1,7 @@
 // NOTE: No IE browsers support
```

Comment:
Not sure why you renamed this, I think it is clearer as a Reset Style since we don't want to use global styles. I think it makes sense to put this as `CSSReset` in components since it functions as one and removing this folder entirely.

Reply:
I might introduce(add-on) a site specific reset rule(unrelated to a browser's behavior. `CSS Reset` is typically used for changing a browser's default css rule), so I renamed it as `GlobalStyle` which is also a common name.